Implantando o teste na PS4eye

pedrombat opened the pull request.

pedrombat committed:
Adicionado documentação em português.
Faltando fazer correções e melhorias

pedrombat committed:
Implantando o teste na PS4eye, falta resolver o problema de reconhecimento da "/dev/video*"

trocolit2 commented on .gitignore:
```diff
@@ -1 +1,2 @@
 build
+*~
```
Tem de parar de usar o gedit e começar a usar o nano ou o vim no terminal.
;)

trocolit2 commented on install.sh:
```diff
@@ -6,9 +6,13 @@
+## kernel_version recebe uma linha do filtro do "uname -r | sed 's/\(\-[0-9]* ..." para saber
+## se o kernel é compativel com a instalação da PS4eye
+
+## se kernel_version for menor que kernel_reference, mostra uma mensagem de error e sai da instalação
```
OK!! Documentação em português está boa, porém tem de explicar um pouco mais em inglês. 
Se tiver duvidas pergunta a Kai, ele ajuda.

Melhore a organização dos comentários. Quando for em inglês coloque uma indicação de qual lingua no inicio, ex:

## EN -> inglês

## PT -> português.

trocolit2 commented on install.sh:
```diff
@@ -22,17 +26,25 @@ echo "Kernel Version $kernel_version"
+
 
 ps4eye_conected=$(lsusb | grep 05a9:0580)
+ps4eye_conected_modified=$(lsusb | grep 05a9:058a)
```
Gostei!! 
Concentrou os comandos em um único lugar, e pode reutilizá-los de forma rápida!

trocolit2 commented on install.sh:
```diff
@@ -22,17 +26,25 @@ echo "Kernel Version $kernel_version"
 
-if ! [ "$ps4eye_conected" ] ; then
+if ! [[ "$ps4eye_conected" || "$ps4eye_conected_modified" ]] ; then
 	echo "ERROR!! PS4EYE NOT CONECTED."
```
Tem uma forma de fazer um #echo como um realce de erro.
Pesquise na net como fazer isso.

trocolit2 commented on install.sh:
```diff
@@ -83,4 +108,25 @@ ls
+
+echo -e "luvcview instaled"
+
+luvcview -d /dev/video1 -i 60 -s 1748x408
```
Essa seleção do vídeo tem de ser automático.
Temos de descobrir como fazer isso.

pedrombat committed:
Adicionado:

- melhora na documentação
- argumento de entrada teste(testar se a ps4eye esta funcionando)
- cores nas saidas de erros e de instalado

pedrombat committed:
Documentação em inglês incluida
